package.json

```diff
@@ -102,24 +102,35 @@
     ],
     "rules": {
       "angular/angularelement": "error",
-      "angular/di": "error",
-      "angular/di-order": "error",
+      "angular/di": "warn",
```

Comment:
Google's standard is to use function syntax rather than array syntax for dependency injection.  However, IIRC function syntax was not always available, and I'm guessing that we kept using array syntax due to momentum.

We believe that eventually we'd like to switch to function syntax to comply with Google's standard.  We should backlog a discussion about this.  @ChristianMurphy mentioned that array syntax is actually useful for minifiers to be able to minify DI variables.